lib/core/src/display/scene.rs

```diff
@@ -375,7 +378,7 @@ impl {
             }
             self.logger.info("Rendering meshes.");
             self.symbols.render(&self.camera);
-//            self.css3d_renderer.render(&self.camera);
+            self.css3d_renderer.render(&self.camera);
```

Comment:
Oh God, I believe ciommenting this was my accidental change. I am so sorry about it. I hope it did not cause any problems and you did not lose too much time tracking it down? :(

Reply:
lol I spent 30m trying to figure that out :P but it's fine.

Reply:
@notdanilo I'm really, really sorry :(